cmd/syncEnumerator.go

```diff
@@ -55,14 +56,15 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 	}
 
 	// TODO: enable symlink support in a future release after evaluating the implications
+	// TODO: Consider passing an errorChannel so that enumeration errors during sync can be conveyed to the caller.
```

Comment:
We are using this error channel to communicate errors while enumeration. We need to add errorChannel for blob in case of sync. This TODO is on us. We take care as part of sync changes.

Reply:
Yes, there will be sync PR. But that will take sometime.